web/.eslintrc.js

```diff
@@ -0,0 +1,104 @@
+/*
+ reference: https://github.com/toshi-toma/eslint-config-airbnb-typescript-prettier
+*/
+module.exports = {
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    sourceType: "module",
+    project: "./tsconfig.json",
+    tsconfigRootDir: "./",
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+  env: {
+    browser: true,
+    es2017: true,
+  },
+  extends: [
+    "airbnb",
+    "airbnb/hooks",
+    "prettier",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:import/typescript",
+  ],
+  plugins: ["react", "jsx-a11y", "import", "prettier", "@typescript-eslint", "simple-import-sort",],
+  ignorePatterns: [".eslintrc.js", "webpack.config.js"],
+  globals: {},
+  rules: {
+    // prettier
+    "prettier/prettier": ["error"],
+
+    // TypeScript
+    "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/ban-ts-ignore": "off",
+    "@typescript-eslint/no-unused-vars": "error",
+    "@typescript-eslint/explicit-member-accessibility": "off",
+    "@typescript-eslint/no-object-literal-type-assertion": "off",
+    "@typescript-eslint/no-shadow": "warn",
+    "@typescript-eslint/ban-types": "warn",
+    "@typescript-eslint/no-explicit-any": "warn",
+    /* 명시적인 return type 작성*/
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+
+    // js
+    "camelcase": "off",
+    "react/require-default-props": "off",
+    "consistent-return": "off",
+    /* redux toolkit immer js를 위한 세팅 */
+    'no-param-reassign': ['error', { props: true, ignorePropertyModificationsFor: ['state'] }],
```

Comment:
p4; 쌍따옴표로 통일하면 좋을 것 같습니다 

```suggestion
    "no-param-reassign": ["error", { props: true, ignorePropertyModificationsFor: ["state"] }],
```

Reply:
https://github.com/banksalad/styleguide/pull/35/commits/74ef932e27e7e476610d6e494518fd3e6d94b702
반영완료